wiki/Guides/Starting_a_Beatmap_Project/id.md

```diff
@@ -0,0 +1,34 @@
+# Cara memulai sebuah proyek beatmap
+
+Perhatian: Artikel ini membahas tentang sub-forum [Beatmap Projects](https://osu.ppy.sh/community/forums/53) dan bagaimana cara menggunakannya. Mohon membaca [aturan forum](https://osu.ppy.sh/community/forums/topics/453937) terlebih dahulu sebelum membuat utas baru.
+
+## Membuat judul utas yang informatif
+
+*Ya, judul utasmu menjadi penentu bagi pengguna osu! yang akan "tertarik" atau "tidak tertarik" pada proyek beatmap-mu, bahkan sebelum mereka masuk pada utas tersebut!*
+
+`[Discography]`/`[Album]`/`[Collab]` harus menjadi prefiks pada judul utas. Utasmu dapat diabaikan atau dipindahkan ke [AW](/wiki/Abandoned_wasteland) jika utasmu tidak memiliki cukup informasi.
+
+**Contoh:**
+
+Cara yang benar:
+
+- `[Album] Pandora Voxx -complete-`
+- `[Discography] Deemo`
+
+Cara yang **salah**:
+
+- `starting a new project, need your help with this album`
+
+## Prefiks mana yang harus saya gunakan?
+
+**Discography:** Daftar album/lagu yang dirilis oleh penyanyi/grup band/komposer/lagu secara keseluruhan.
+
+- Contoh: [Duca](https://osu.ppy.sh/community/forums/topics/98205), [Linkin Park](https://osu.ppy.sh/community/forums/topics/121592), [Cytus](https://osu.ppy.sh/community/forums/topics/177446)
+
+**Album:** Daftar lagu pada sebuah album.
+
+- Contoh: [Anamanaguchi - Scott Pilgrim vs The World (The Game)](https://osu.ppy.sh/community/forums/topics/37908), [Daft Punk - Random Access Memories](https://osu.ppy.sh/community/forums/topics/132592)
+
+**Collab:** Membuat [proyek *group-mapping*](/wiki/Beatmap/Beatmap_collaborations) pada lagu tertentu. Dapat berkisar dari permintaan pembuatan [storyboard](/wiki/Storyboard) hingga pembagian mapping dari sebuah [map marathon](/wiki/Beatmap/Marathon). [Referensi](https://osu.ppy.sh/community/forums/topics/35841)
+
+- Contoh: [Within Temptation ~ The Unforgiving](https://osu.ppy.sh/community/forums/topics/56629)
```

Comment:
ignore the hyphen if it's supposed to be a tilde in indonesian

```suggestion
- Contoh: [Within Temptation - The Unforgiving](https://osu.ppy.sh/beatmapsets/29157)
```

Reply:
Tilde symbol is from forum post title but if the link changed to beatmap page, it should changed to hyphen.